.travis.yml

```diff
@@ -38,4 +38,5 @@ addons:
     - sourceline: 'ppa:chris-lea/libsodium'
     packages:
     - libsodium-dev
+    - libgit2-dev
```

Comment:
This is because the new {usethis} depends on {gert} which needs `libgit2-dev` installed on the system. 

I don't know if this will be needed on Actions or not, we'll find out!